app/destination/page.js

```diff
@@ -1,54 +1,68 @@
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+import { useEffect, useState } from "react";
+import { PlanetWishlistItem } from "./components/planet_wishlist/page";
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import { PlanetCard } from "./components/planet_card/planetCard";
 
+const planets = [
+  {
+    name: "EUROPA",
+    description: `Europa, one of Jupiter’s moons, is an icy world with a hidden ocean beneath its surface. This mysterious moon is a prime candidate for the search for extraterrestrial life, making it a thrilling destination for space explorers.`,
+    thumbnail: "/destination/image-europa.png",
+  },
+  {
+    name: "Mars",
+    description: `Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth.`,
+    thumbnail: "/destination/image-mars.png",
+  },
+  {
+    name: "Moon",
+    description: `Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers.`,
+    thumbnail: "/destination/image-moon.png",
+  },
+  {
+    name: "Titan",
+    description: `Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.`,
+    thumbnail: "/destination/image-titan.png",
+  },
+];
 
 export const Destinations = () => {
   const [selectedPlanets, onAddPlanet] = useState([]);
 
   let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  let numberOfPlanets = selectedPlanets.length;
 
   const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    // Implement this function
-    // If you press the "ADD PLANET" the selected planet should display "SELECTED"
-    // And the counter should update, how many planets are selected (numberOfPlanets)
-    console.log(`You seleceted the following planet: ${name}, with the index of ${index}`);
-  }
+    onAddPlanet((selectedPlanets) => {
+      if (selectedPlanets.includes(name)) {
+        return selectedPlanets.filter((planet) => planet !== name);
+      } else {
+        return [...selectedPlanets, name];
+      }
+    });
+
+    console.log(
+      `You selected the following planet: ${name}, with the index of ${index}`
+    );
+    isPlanetSelected = (name) => selectedPlanets.includes(name);
+    console.log(isPlanetSelected);
```

Comment:
Usually we delete the debugging code before we create the PR. 😄

Reply:
Thanks, I have deleted the console now.